app/models/Level.js

```diff
@@ -326,6 +326,26 @@ module.exports = (Level = (function () {
               heroThangType = ThangTypeConstants.heroes[juniorHeroReplacement]
             }
             levelThang.thangType = heroThangType
+
+            // merges hero for junior level
+            if (this.get('product', true) === 'codecombat-junior') {
+              const thangType = thangTypesByOriginal[levelThang.thangType]
+              const configs = {}
+              for (const thangComponent of Array.from(levelThang.components)) {
```

Comment:
`Array.from` is not necessary, these must already be arrays. We only have `Array.from` on many other places, because decaffeination from coffeescript left them there.